pkg/client/cluster.go

```diff
@@ -18,15 +17,63 @@ func (c *client) Ping(vsn string) error {
 	return err
 }
 
-func (c *client) RegisterRuntimeServices(svcs map[string]string, serviceId *string) error {
+func initLayouts(layouts *console.OperationalLayoutAttributes) *console.OperationalLayoutAttributes {
+	if layouts == nil {
+		return &console.OperationalLayoutAttributes{
+			Namespaces: &console.ClusterNamespacesAttributes{},
+		}
+	}
+	return layouts
+}
+
+func appendUniqueExternalDNSNamespace(slice []*string, newValue *string) []*string {
```

Comment:
lets use a set to build this, then convert it to an array when presenting it to the console api

Reply:
done